handlers/config.go

```diff
@@ -7,10 +7,12 @@ import (
 
 // Container Struct to hold the configuration for Job Container
 type Container struct {
-	Name       string   `json:"name"`
-	Image      string   `json:"image"`
-	PullPolicy string   `json:"pull_policy"`
-	Env        []string `json:"env"`
+	Name        string   `json:"name"`
+	Image       string   `json:"image"`
+	PullPolicy  string   `json:"pull_policy"`
+	Env         []string `json:"env"`
+	CPULimit    string   `json:"cpu_limit"`
```

Comment:
Can I suggest we switch to dash `-` to match Hatchery? https://github.com/uc-cdis/hatchery/blob/master/src/handlers/config.go#L13
or change Hatchery, but lets not drive ourselves crazy with different variations for different services.

Reply:
👍 Nice, haven't check `hatchery` yet, it now match its configuration.